lib/Frontend/ParseableInterfaceModuleLoader.cpp

```diff
@@ -582,6 +582,8 @@ class swift::ParseableInterfaceBuilder {
       std::string OutPathStr = OutPath;
       SerializationOpts.OutputPath = OutPathStr.c_str();
       SerializationOpts.ModuleLinkName = FEOpts.ModuleLinkName;
+      SerializationOpts.ParseableInterface =
+          FEOpts.InputsAndOutputs.AllInputs[0].file();
```

Comment:
This should already be in scope as `InPath`, I think you can use that.

Reply:
Done!